combat-trainer.lic

```diff
@@ -2183,6 +2193,33 @@ class GameState
     end
   end
 
+  def tactics_analyze?
```

Comment:
I didn't write the `perform_analyze?` method nor do I have a barb-- so I cannot guess why it lacks that case.

The differences are between the barb version and non version are small. True. However, those differences can cause a lot of trouble if you try to make it universally applicable/usable to non-barbs:

- It expects two arguments to be passed through it.
- Has a return based on IF-related crap.
- Has a return based on expertise crap.

Additional reasons were:
- I don't have a barb to test edits to that method to ensure I didn't break it for barbarians.
- I initially expected to use cases for every magnitude of _<size> opening_ gained by analyze.
- I just wanted it to lock tactics quickly and it does!